app/src/main/res/values/strings.xml

```diff
@@ -113,7 +113,8 @@
     <string name="progress_km">%1$dkm</string>
     <string name="selected_age_range">%1$d - %2$d</string>
     <string name="not_logged_in">NOT LOGGED-IN</string>
-    <string name="logout_error">LOGOUT ERROR</string>
+    <string name="logout_error">Received error while trying to log out </string>
```

Comment:
Nitpicking to an extreme, is the little space in the end needed :D ?

Reply:
Damn I missed it thanks x)